pa_config/rca_master.conf

```diff
@@ -97,11 +97,21 @@
     "ClusterTemperatureRca"
   ],
 
+  // Default Values are added in the config specific file as well
+  // Modify them as well whenever changing the default values (DeciderConfig.java)
   "decider-config-settings": {
+    // Decreasing order of priority for the type of workload we can expect on the cluster.
+    "workload-type": {
+      "priority-order": ["ingest", "search"]
+    },
+    // Decreasing order of priority for the type of cache which is expected to be consumed more.
+    "cache-type": {
+      "priority-order": ["read-heavy", "write-heavy"]
+    },
     // cache decider - Needs to be updated as per the performance test results
-    "cache-decider-config": {
-      "field-data-cache-upper-bound" : 0.4,
-      "shard-request-cache-upper-bound" : 0.05
+    "cache-bounds": {
+      "field-data-cache-upper-bound" : 10.4,
+      "shard-request-cache-upper-bound" : 10.05
```

Comment:
What are your thoughts on having one more level. Like all cache related config bounded together. Pro might be it becomes more manageable.

Reply:
We don't anticipate any more configs just now. So I will be more tilted to keep it just like this. I agree if too many configs are available then it would be nice to club similar of them together.